WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyRingItemViewModel.cs

```diff
@@ -28,6 +28,8 @@ public PrivacyRingItemViewModel(PrivacyRingViewModel parent, SmartCoin coin, dou
 		Unconfirmed = !coin.Confirmed;
 		Confirmations = coin.GetConfirmations();
 
+		PrivacyLevelText = GetPrivacyLevelDescription();
```

Comment:
I think you can apply this even if the segments are pockets. That would also eliminate the null warning.